document-legacy/src/layers/style/mod.rs

```diff
@@ -58,6 +58,13 @@ pub enum GradientType {
 	Radial,
 }
 
+#[derive(Default, PartialEq, Eq, Clone, Copy, Debug, Hash, Serialize, Deserialize, specta::Type)]
+pub enum SelectedType {
```

Comment:
I chose this file because I was using the tool_messages/gradient_tool.rs as a reference and in the same file there is a similar enum but for the gradient tool. I can move it somewhere else if there's a better spot for it.